server/routes.go

```diff
@@ -12,6 +12,12 @@ import (
 	"github.com/sirupsen/logrus"
 )
 
+type ScalerFunc func(ctx context.Context) error
+
+func EmptyScalerFunc() ScalerFunc {
+	return func(ctx context.Context) error { return nil }
+}
```

Comment:
Could simplify further with

```suggestion
var EmptyScalerFunc = func(ctx context.Context) error {return nil}
```